app/services/export/s3_uploader.rb

```diff
@@ -34,14 +34,7 @@ def upload
       )
       raise "Unexpected response." unless response&.etag
 
-      if use_public_bucket
-        OpenStruct.new(
-          url: bucket.object(filename).public_url,
-          timestamped_filename: filename
-        )
-      else
-        OpenStruct.new(timestamped_filename: filename)
-      end
+      OpenStruct.new(timestamped_filename: filename)
```

Comment:
I would really like to see a clearer commit message for this, I feel like it is less about the expectations and more about removing the need for `use_public_bucket` why are we doing that?

Reply:
It's a fair question, I was a bit lax in retrieving the history, but now I have. (And found a comment from Ynda suggesting we could remove `use_public_bucket` already, back then 🙈 )